<launch> 
    <node name="g4t1" pkg="component" type="g4t1" output="screen" />

    <param name="frequency" value="50" /> <!-- 1 Hz  -->

    <param name="connect" value="false" type="bool" />
    <!-- Firebase Realtime Database -->
    <param name="db_url" value="https://my-project-1516369881504.firebaseio.com" />
    <param name="session" value="25" type="int" />

</launch>